src/passes/GenerateDynCalls.cpp

```diff
@@ -127,9 +127,9 @@ void GenerateDynCalls::generateDynCallThunk(Signature sig) {
   for (const auto& param : sig.params) {
     args.push_back(builder.makeLocalGet(++i, param));
   }
-  // FIXME: Should the existence of a table be ensured here? i.e. create one if
-  // there is none?
-  assert(wasm->tables.size() > 0);
+  if (wasm->tables.empty()) {
+    wasm->addTable(Builder::makeTable(Name::fromInt(0)));
```

Comment:
Most likely you we want to create a table import here.. to mimic the older linker behaviour (which was to create a an unused import).

Reply:
Ah, thanks, that makes sense. When I add an import with the exact same name as the backend would, it works.